### ISSUE TITLE
Package amdgpu-pro-opencl-icd must provide opencl-icd-loader

### ISSUE THREAD

**Issue by ranomier** [corngood/archlinux-amdgpu#18]
Im not sure but i tried a update with `pacaur -Syu` and got this:

```
ocl-icd and amdgpu-pro-libopencl-dev are in conflict (libcl). Remove amdgpu-pro-libopencl-dev? [y/N]
unresolvable package conflicts detected
failed to prepare transaction (conflicting dependencies)
ocl-icd and amdgpu-pro-libopencl-dev are in conflict
```

Package ocl-icd provides:
`Provides        : libcl  opencl-icd-loader`

Package amdgpu-pro-opencl-icd provides nothing.

**Comment by maxcr** [comment]
I wrote a one liner because once its installed its worse than herpes to remove.

sudo pacman -Qe | grep amdgpu | cut -f1 -d ' ' | sudo xargs pacman -Rnsdd --noconfirm

You may want to change it amd-pro instead of amdgpu

Several other packages need to be forcefully removed and reinstalled using this method such as opencl-headers ocl-icd libclc. Once that's done you should be golden. That's how I fixed my install at least.

**Comment by corngood** [comment]
This should be fixed in #28.  If not, please reopen.
